### PR TITLE
veeresh9148/angulerproject

### PR DESCRIPTION
**Pull request by veeresh9148**
Changes by Veeresh

### CODE REVIEW

File: Veeresh/HTML5/Assignment1.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            }
+            
+        </style>
+    </head>
+    <body>
+        <h3>Anguler Form</h3>
+        <div id="form">
+        <form action="">
+            <label id="label"> Name </label><br>
+           <input type="text" name="name" id="text" placeholder="Enter Your Name" autocomplete="off"><br><br>
```
Comment (itssinghashish) [inline Veeresh/HTML5/Assignment1.html:85]: Please give proper id's. some meaningful id. Also check the same for other id's

**Review by itssinghashish — CHANGES_REQUESTED** [review]
Please change the input id's

File: Veeresh/HTML5/Assignment1.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+        }
+    </style>
+</head>
+
+<body>
+    <h3>Anguler Form</h3>
+    <div id="form">
+        <form action="">
+            <label for="name" id="label_name"> Name </label><br>
+            <input type="text" name="name" id="name" placeholder="Enter Your Name" autocomplete="off"><br><br>
```
Comment (itssinghashish) [inline Veeresh/HTML5/Assignment1.html:97]: change as inp_name

File: Veeresh/HTML5/Assignment1.html [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+            <label for="email" id="label_email"> Email </label><br>
+            <input type="email" name="name" id="email" placeholder="Enter Your Email" autocomplete="off"><br><br>
```
Comment (itssinghashish) [inline Veeresh/HTML5/Assignment1.html:99]: change as inp_email. Similarly change for others. Try to standardize the naming convention.

**Review by itssinghashish — CHANGES_REQUESTED** [review]
please check my inline comments

**Review by itssinghashish — COMMENTED** [review]
looks good hence approving

**Review by rajukt15 — APPROVED** [review]

**Review by itssinghashish — APPROVED** [review]

**Pull request merged**